policy/feature-flags.md

```diff
@@ -6,63 +6,137 @@ Product and release need to track bugs whose visibility is controlled through a
+- The bug for the feature should be moved to RESOLVED VERIFIED status
+- The bug should be enabled in Beta once Release Management approves
+- the `behind-pref` flag is updated to `betaNN+1` 
 
-By default, the preference enabling the feature **should not** be set.
+The feature now *rides the trains* to release, and on merge day, `behind-pref` is set to `nightlyNN`. The bug is then considered completed. 
 
-A patch lands in mozilla-central with the code for the feature controlled by the preference, then:
+When the next ESR is released, the `behind-pref-esr` field should be set to the version where it was relased. 
 
-*   The bug's status is moved to RESOLVED and resolution to FIXED.
-*   QA agrees to take on verfication, setting `qe-verify` to `+`
-*   The feature is tested with and without the flag set and confirmed to work
-*   QA moveds the bug's resolution to VERIFIED
-*   Otherwise the patch is backed out to be landed again later, and the bug REOPENED
+### Questions 
 
+#### What if we turn off the feature in the main release?
 
-Once the bug for the feature has reached RESOLVED VERIFIED, then the lead for the feature which may be an engineer, a program manager, or a product manager must confirm with the Nightly Release Manager, and then enable it.
+The main bug's `behind-pref` value should be reset to the releases it's still on, `betaNN+1` or `nightlyNN+2`; or to `off`, and the bug's status set to REOPENED.
 
-*   The patch to flip the pref on should be attached to the "Make Tabby Cats the new default tab experience" bug
+The bug to turn off the feature must be a dependency of the main bug.
 
-If a decision is made to uplift the feature to another release, then RelMan and product must decide when to enable the feature. If they do decide to enable it then:
+#### What if we want to hold a feature over a release cycle and not promote it?
 
-*   The patch to turn the pref on which has been attached to the "Make Tabby Cats the new default tab experience" bug should be requested to be uplifted to the appropriate train
+On merge day, the `behind-pref` flag would change from the previous to the new Beta or Nightly value, and remain preffed off in other versions.
 
-Once we reach code freeze for the the new release, RelMan and product must decide if the feature goes out with the flag turned on, then:
+#### What if I want to enable parts of my feature in Nightly?
 
-*   A bug is filed to turn off the feature. The bug **must** be marked as blocking release.
+If your feature is incomplete, but some functionality is avalable, then mark `behind-pref` as `nightlyNN+2`. Do not request `qe-verify` until the feature is complete.
 
-If the release goes out with the preference for the feature turned off, another bug **must** be filed to turn the preference on.
+If you intend to release functionality over a number of releases, then each set of functionality should be treated as a separate `[meta]` bug subject to the process described in this document.
```

Comment:
Replace "[meta]" with "feature/meta"

Btw. meta is a keyword whereas square brackets are usually for whiteboard fields.